components/file_attachment_list/file_attachment_list.jsx

```diff
@@ -91,17 +91,14 @@ export default class FileAttachmentList extends React.Component {
         }
 
         const postFiles = [];
-        let sortedFileInfos = [];
-
         if (fileInfos && fileInfos.length > 0) {
-            sortedFileInfos = fileInfos.sort((a, b) => a.create_at - b.create_at);
-            for (let i = 0; i < Math.min(sortedFileInfos.length, Constants.MAX_DISPLAY_FILES); i++) {
-                const fileInfo = sortedFileInfos[i];
+            for (let i = 0; i < Math.min(fileInfos.length, Constants.MAX_DISPLAY_FILES); i++) {
```

Comment:
Why do we do this minimum here? When would `fileInfos.length` ever be > `Constants.MAX_DISPLAY_FILES`? And if it is, why would we hide a preview from the user?

Reply:
You're right.  I've verified it and it's not necessary.  File upload limit is handled by `MAX_UPLOAD_FILES:5` constant.